<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>INVENTIVA STOCK</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/python.png</normaloff>../assets/python.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century-ghotic;
}

QWidget#background{
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(20, 20, 20), stop:1 rgb(80, 80, 80));
}

QFrame#frame_main{
	background-color: rgb(20, 20, 20);
	border-radius: 15px;
}

QFrame#edit_stock_frame {
	background-color: rgb(120, 120, 120);
	border-radius: 8px;
}

QLabel{
	color: rgb(230, 230, 230);
}

QLineEdit{
	background: transparent;
	border: none;
	border-bottom: 1px solid white;
	color: white;
	font-size: 14px;
}

QPushButton{
	background-color: rgb(152, 152, 152);
	font-weight: bold;
	font-size: 20px;
	border:none;
	border-radius: 8px;
}

QPushButton:hover {
    background-color: rgb(190, 190, 190);
}

QPushButton:pressed {
    background-color: rgb(120, 120, 120);
}

QMessageBox{
	color: rgb(0, 0, 0);
}

QHeaderView::section {
    font-weight: bold;
    background-color: transparent;
	color: rgb(230, 230, 230);
}

QTableWidget {
    background-color: transparent;
	color: rgb(230, 230, 230);
	gridline-color: #ffffff;
}

QHeaderView::section {
	border: 1px solid #ffffff;
}

QTableWidget::item {
    border-right: 8px solid rgb(20, 20, 20);
	border-left: 8px solid rgb(20, 20, 20);
}

QTableWidget QHeaderView {
    background-color: transparent;
}

QTableWidget QTableCornerButton::section {
    background-color: transparent;
}

QTableWidget QScrollBar:vertical {
    border: none;
    background: rgb(152, 152, 152);
	width: 15px;
}

QTableWidget QScrollBar::add-page:vertical, QTableWidget QScrollBar::sub-page:vertical {
    background: rgb(152, 152, 152);
}
QTableWidget QScrollBar::handle:vertical {
    background: rgb(44, 44, 44);
	margin: 2px 2.5px;
	border-radius: 4px;
}

QTableWidget QScrollBar::sub-line:vertical, QTableWidget QScrollBar::add-line:vertical {
    height: 0px;
    border: none;
    background: none;
}

QFrame#edit_stock_frame QLabel {
	color: black;
}

QLabel#lbl_id_nombre {
	font-family: century-ghotic;
    font-size: 18px;
	color: black;
}

QPushButton#btn_apply{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	font-weight: bold;
	font-size: 16px;
	border:none;
	border-radius: 5px;
}

QPushButton#btn_apply:hover {
    background-color: rgb(50, 50, 50);
}

QPushButton#btn_apply:pressed {
    background-color: rgb(150, 150, 150);
}

QSpinBox {
	font-size: 18px;
    background-color: rgb(120, 120, 120);
    color: #000000;
    border: none;
    padding: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="background">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_producto">
      <property name="minimumSize">
       <size>
        <width>1366</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1633</width>
        <height>768</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_main">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>1331</width>
         <height>731</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget QScrollBar::sub-line:vertical, QTableWidget QScrollBar::add-line:vertical {
    height: 0px;
    border: none;
    background: transparent;
}

QTableWidget QScrollBar::up-arrow:vertical, QTableWidget QScrollBar::down-arrow:vertical {
    width: 0px;
    height: 0px;
    background: transparent;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableWidget" name="tblProducto">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>1261</width>
          <height>521</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descripción</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Estado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Stock</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Peso</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ingreso</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Subfamilia</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Familia</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Línea</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="txtBuscarProducto">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Buscar...</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_title">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Inventiva Stock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="edit_stock_frame">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>90</y>
          <width>841</width>
          <height>71</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="lbl_id_nombre">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>531</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ID12345: NOMBRE PRODUCTO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_stock">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>0</y>
           <width>61</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Stock:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spx_stock">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_apply">
         <property name="geometry">
          <rect>
           <x>690</x>
           <y>20</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Aplicar</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableWidget" name="tblProducto_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>1261</width>
          <height>521</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Contraseña</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Apellidos</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombres</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Identificación</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dirección</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ubigeo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Teléfono</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Correo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fecha nac.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sexo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sueldo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Turno</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>1162</x>
          <y>0</y>
          <width>161</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>INVENTARISTA</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>0</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PRODUCTO</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>agregar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>eliminar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>editar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
